cmd/tailscale/cli/web.go

```diff
@@ -375,7 +375,7 @@ func webHandler(w http.ResponseWriter, r *http.Request) {
 			data.AdvertiseExitNode = true
 		} else {
 			if data.AdvertiseRoutes != "" {
-				data.AdvertiseRoutes = ","
+				data.AdvertiseRoutes += ","
```

Comment:
actually! we have a better API for this now.

See 2cfc96aa901433ae407883ce8856244b3a196a49

Reply:
oh, but this is some weird template thing